src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/signup/AlmostTherePageObject.java

```diff
@@ -31,20 +31,31 @@ public void verifyAlmostTherePage() {
     waitForElementByElement(changeMyEmail);
   }
 
-  private String getActivationLinkFromMail(String email, String password) {
+  private String getActivationLinkFromMail(String email, String password, String language) {
+    String mailSubject;
+    if (language == "ja") {
```

Comment:
String is an object, so for String comparison please use:
language.equals("ja")